app/views/brexit_checker/_action_list.html.erb

```diff
@@ -16,22 +16,32 @@
           <%= action.title %>
         <% end %>
       </p>
+      <% if action.lead_time.present? %>
+        <div class="govuk-body govuk-!-font-weight-bold brexit-checker-result__leadtime">
```

Comment:
I don't think `brexit-checker-result__leadtime` is doing anything - there are no associated styles.

Reply:
nope, wasn't - great catch